Zend/tests/accessors/backing_prop.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+Backing property
+--FILE--
+<?php
+
+class Test {
+    public int $prop {
+        get { return $this->prop * 2; }
```

Comment:
If there is no use of a backing property, is that property still added to that object?

Like when a different property is used in getter/setter:
```
        get { return $this->different; }
        set { $this->different = $value; }
```

Reply:
Currently, the backing property is always added, but it shouldn't be. Unfortunately this is much harder than it sounds, as it's still possible to use the property recursively, in which case this should result in a dynamic rather than pre-allocated property. But that breaks some assumptions.

I've been wondering if I can somehow make this a hard either/or and don't allow use of the "backing property" at all when implementing accessors manually (only use it for automatically implemented ones).

Reply:
This is now implemented. No backing prop for explicit accessors, and recursion in accessors is forbidden.